scripts/eqtl_hail_batch/README.md

```diff
@@ -0,0 +1,45 @@
+# Run hail batch job to generate list of eQTLs
+
+This runs a Hail batch script in order to generate a list of eQTLs from scRNA-seq expression. This code was taken from Seyhan Yazar from Joseph Powell's group at the Garvan-Weizmann Centre for Cellular Genomics, then converted into Python/hail batch. To run, use conda to install the analysis-runner. For the first round of eQTLs, execute the following command for running one cell type and chromosome (B_intermediate cells and chromosome 22 are shown here as an example):
```

Comment:
Could you link to that original codebase here?

Reply:
Great idea! The publicly-available code wasn't available when I when I was writing this, but now that the OneK1K paper is out, it is. I'll add that link in :)